jdat_notebooks/background_estimation_imaging/Imaging Sky Background Estimation.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    mask.sum()
</pre><p>the sum isn't discussed anywhere.  Is it needed or was just for debugging?  If it is needed, maybe it would be better to wrap in a print statement.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
Harry wrote this. Let's keep it for now given the time crunch.

Reply:
works for me!